easybuild/easyconfigs/b/BamTools/BamTools-2.5.0-foss-2016b.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'CMakeMake'
+
+name = 'BamTools'
+version = '2.5.0'
+
+homepage = 'https://github.com/pezmaster31/%(namelower)s'
+description = "BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files."
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/pezmaster31/%(namelower)s/archive']
+sources = ['v%(version)s.tar.gz']
+
+checksums = ['85e02e04998a67cbda7ab68cdab36cee133db024e814b34e06bb617b627caf9c']
+
+builddependencies =[('CMake', '3.7.2')]
```

Comment:
@SethosII style check fails because of missing space after `=` here...

Reply:
@boegel I changed this (with a force push).